wandb/sdk/artifacts/storage_policies/wandb_storage_policy.py

```diff
@@ -399,5 +400,9 @@ def _write_cache(self, entry: "ArtifactManifestEntry") -> None:
             try:
                 with cache_open("wb") as f, open(entry.local_path, "rb") as src:
                     shutil.copyfileobj(src, f)
+                if entry.policy == "mutable":
```

Comment:
I think that instead of checking the policy here we should just check the location of `local_path` to see if it's in the staging directory--if it is, we always delete it.

Reply:
oh wait, but since we're copying them to the cache, i guess we could just move them from there too 🤦 . nevermind - i'll revert this back to the original change of just checking for the staging dir prefix in `entry.local_path`.